frontend/src/scenes/trends/personsModalLogic.ts

```diff
@@ -145,8 +146,16 @@ export const personsModalLogic = kea<personsModalLogicType<PersonModalParams>>({
                 const filterParams = parsePeopleParams({ label, action, date_from, date_to, breakdown_value }, filters)
                 actions.setPeople(tempPeople)
                 people = await api.get(`api/person/stickiness/?${filterParams}${searchTermParam}`)
-            } else if (funnelStep) {
-                const params = { ...funnelLogic().values.filters, funnel_step: funnelStep }
+            } else if (funnelStep || filters.funnel_viz_type === FunnelVizType.Trends) {
+                let params
+                if (filters.funnel_viz_type === FunnelVizType.Trends) {
+                    // funnel trends
+                    const entrance_period_start = dayjs(date_from).format('YYYY-MM-DD HH:mm:ss')
+                    params = { ...funnelLogic().values.filters, entrance_period_start, drop_off: false }
+                } else {
+                    // regular funnel steps
+                    params = { ...funnelLogic().values.filters, funnel_step: funnelStep }
+                }
```

Comment:
This part is a bit strange as it introduces very tight coupling from the persons modal to the `funnelLogic`. Is this coupling needed? Doesn't the local `filters` value already contain what is needed?

Reply:
yep you are correct 👍